### PR TITLE
Add better documentation and update polyfill to mirror mdn polyfill again

### PR DESCRIPTION
**Pull request by christiansany**
Provides a better documentation on how to use this polyfill.

Additionally, I am the writer of the polyfill for MDN and i would be very pleased to be an official contributor to this package.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,31 @@
...
 
-MDN polyfill for the `NodeList.forEach` method.
-
-It polyfills `forEach` support for `NodeList` objects (a common result of `document.querySelectorAll`).
+Provides a polyfill for [Nodelist.prototype.forEach()](https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach) to all Browsers supporting [ES5](https://caniuse.com/#search=es5).
 
 ## Native support
 
-Chrome 51, Firefox 50, Opera 38, Safari 10
+Chrome 51, Firefox 50, Opera 38, Safari 10<br>
```
Comment (imagitama) [inline README.md]: In Markdown you can use double space for new lines (I should've done this - good catch)

File: README.md [hunk 2]
```diff
@@ -1,17 +1,31 @@
...
+
+## Import
 
-Android Browser none, IE Mobile none, IE none
+```JavaScript
+// CommonJS
+require('nodelist-foreach-polyfill');
 
-See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach)
+// ES6 Modules
```
Comment (imagitama) [inline README.md]: Should/could this be ES6 modules/TypeScript?

**Review by imagitama — COMMENTED** [review]
Thanks for the pull request! I've left some comments. I'm happy to merge once resolved. 👍

**Comment by christiansany** [comment]
Hi :)
I've updated the Readme.md file

**Comment by christiansany** [comment]
Are you integrating it, or are there further things to add? :)

**Comment by imagitama** [comment]
Hey sorry about that! I've published your changes as 1.2.0. :)

**Comment by christiansany** [comment]
Cool 👍

**Pull request merged**